bin/ream/src/cli/mod.rs

```diff
@@ -86,6 +86,25 @@ pub struct NodeConfig {
         help = "One or more comma-delimited base64-encoded ENR's of peers to initially connect to. Use 'default' to use the default bootnodes for the network. Use 'none' to disable bootnodes."
     )]
     pub bootnodes: Bootnodes,
+
+    #[arg(
+        long = "checkpoint-sync-url",
+        help = "Trusted RPC URL to initiate Checkpoint Sync."
+    )]
+    pub checkpoint_sync_url: Option<String>,
+
+    #[arg(
+        long = "purge-db",
+        help = "Purges the database.",
+        action = clap::ArgAction::SetTrue
+    )]
+    pub purge_db: bool,
```

Comment:
```suggestion
    #[arg(
        long = "purge-db",
        help = "Purges the database.",
    )]
    pub purge_db: bool,
```

we don't need to define this action it is infered